mxcube3/routes/SampleCentring.py

```diff
@@ -657,20 +655,21 @@ def wait_for_centring_finishes(*args, **kwargs):
     centred point.
     """
     centring_status = args[1]
-
     # we do not send/save any centring data if there is no sample
     # to avoid the 2d centring when no sample is mounted
     if scutils.get_current_sample().get('sampleID', '') == '':
         return
-    try:
```

Comment:
Yes indeed, cant really tell how it was before but I imagine that 

    motor_positions = mxcube.diffractometer.centringStatus["motors"]

used to retrieve the centering status and the if statement maybe checked motor_positions[valid], but then someone passed centering status as an argument and this block became redundant.

So the commented block can be removed

Reply:
okey, then I was not crazy...